labconnect/main/routes.py

```diff
@@ -21,16 +22,109 @@
     RPIDepartments,
     RPISchools,
     User,
+    RecommendsCourses,
+    RecommendsClassYears,
 )
 
 from . import main_blueprint
 
 
+def packageOpportunity(opportunityInfo, professorInfo):
```

Comment:
can you put any opportunity routes into the opportunity routes file

Reply:
Yes. Ill try to that. I don't know how to commit changes to a pull request though which is the issue. If I'm not able, can I try to pull request first and then immediately drop another pr soon for organizing those?

Reply:
you can just pull then push new changes to the same branch